diff -u -r -N owncloud/apps/files_sharing/css/sharing.css owncloud/apps/files_sharing/css/sharing.css
--- owncloud/apps/files_sharing/css/sharing.css	2012-01-29 14:50:19.000000000 +0100
+++ owncloud/apps/files_sharing/css/sharing.css	2012-03-28 11:45:02.000000000 +0200
@@ -5,4 +5,7 @@
 #shared_list { padding:0.5em; list-style-type: none; }
 #public { border-top:1px solid #ddd; padding-top:0.5em; }
 a.unshare { float:right; display:inline; margin:0 .5em; padding:.3em .3em 0 .3em !important; opacity:.5; }
-a.unshare:hover { opacity:1; }
\ Kein Zeilenumbruch am Dateiende.
+a.unshare:hover { opacity:1; }
+
+div.actionsSharing { font-weight: bold; padding-left: .65em; padding-top: .65em; color: #555; }
+table td.source, table td.uid_shared_with, table td.permissions { padding: .5em ; }
\ Kein Zeilenumbruch am Dateiende.
diff -u -r -N owncloud/apps/files_sharing/list.php owncloud/apps/files_sharing/list.php
--- owncloud/apps/files_sharing/list.php	2012-01-29 14:50:19.000000000 +0100
+++ owncloud/apps/files_sharing/list.php	2012-03-28 11:43:31.000000000 +0200
@@ -29,6 +29,7 @@
 OC_App::setActiveNavigationEntry("files_sharing_list");
 
 OC_Util::addScript("files_sharing", "list");
+OC_Util::addStyle("files", "files");
 
 $tmpl = new OC_Template("files_sharing", "list", "user");
 $tmpl->assign("shared_items", OC_Share::getMySharedItems());
diff -u -r -N owncloud/apps/files_sharing/templates/list.php owncloud/apps/files_sharing/templates/list.php
--- owncloud/apps/files_sharing/templates/list.php	2012-01-29 14:50:19.000000000 +0100
+++ owncloud/apps/files_sharing/templates/list.php	2012-03-28 11:44:11.000000000 +0200
@@ -1,5 +1,10 @@
-<fieldset>
-	<legend><?php echo $l->t('Your Shared Files');?></legend>
+<div id="controls">
+	<div class="actionsSharing">
+		<?php echo $l->t('My shared files'); ?>
+	</div>
+</div>
+
+<div id='list'>
 	<table id="itemlist">
 		<thead>
 			<tr>
@@ -14,17 +19,17 @@
 					<td class="source"><?php echo substr($item['source'], strlen("/".$_SESSION['user_id']."/files/"));?></td>
 					<td class="uid_shared_with"><?php echo $item['uid_shared_with'];?></td>
 					<td class="permissions"><?php echo $l->t('Read'); echo($item['permissions'] & OC_SHARE::WRITE ? ", ".$l->t('Edit') : ""); echo($item['permissions'] & OC_SHARE::DELETE ? ", ".$l->t('Delete') : "");?></td>
-					<td><button class="delete" data-source="<?php echo $item['source'];?>" data-uid_shared_with="<?php echo $item['uid_shared_with'];?>"><?php echo $l->t('Delete');?></button></td>
+<!--					<td><button class="delete" data-source="<?php echo $item['source'];?>" data-uid_shared_with="<?php echo $item['uid_shared_with'];?>"><?php echo $l->t('Delete');?></button></td> //-->
 				</tr>
 			<?php endforeach;?>
-			<tr id="share_item_row">
+<!--			<tr id="share_item_row">
 				<form action="#" id="share_item">
 					<td class="source"><input placeholder="Item" id="source" /></td>
 					<td class="uid_shared_with"><input placeholder="Share With" id="uid_shared_with" /></td>
 					<td class="permissions"><input placeholder="Permissions" id="permissions" /></td>
 					<td><input type="submit" value="Share" /></td>
 				</form>
-			</tr>
+			</tr> //-->
 		</tbody>
 	</table>
-</fieldset>
+</div>
